Build failure: freeplane

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

1. Ensure you are on the unstable branch of Nixpkgs.
2. Run `nix shell -p freeplane "freeplane"` and watch it try to build.
3. See the build error.

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/296879207

### Relevant log output

```console
FAILURE: Build failed with an exception.

* Where:
Build file '/build/source/build.gradle' line: 13

* What went wrong:
An exception occurred applying plugin request [id: 'com.netflix.nebula.ospackage', version: '11.11.1']
> Failed to apply plugin class 'com.netflix.gradle.plugins.packaging.SystemPackagingBasePlugin'.
   > Could not create an instance of type com.netflix.gradle.plugins.packaging.ProjectPackagingExtension.
      > org.gradle.api.tasks.util.internal.PatternSets$PatternSetFactory
```

### Additional context

Likely a problem with updated dependencies, as freeplane 1.12.8 built fine in the past.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.28, NixOS, 25.05 (Warbler), 25.05.20250510.d89fc19`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.3`
 - channels(root): `"home-manager, nixos-24.05"`
 - nixpkgs: `/nix/store/ccfwxygjrarahgfv5865x2f828sjr5h0-source`

### Notify maintainers

@charles-dyfis-net

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/fr/freeplane/package.nix
+++ b/pkgs/by-name/fr/freeplane/package.nix
@@ -13,7 +13,7 @@
 
 let
   pname = "freeplane";
-  version = "1.12.8";
+  version = "1.12.10";
 
   jdk = jdk17;
   gradle = gradle_8;
@@ -22,7 +22,7 @@ let
     owner = "freeplane";
     repo = "freeplane";
     rev = "release-${version}";
-    hash = "sha256-yzjzaobXuQH8CHz183ditL2LsCXU5xLh4+3El4Ffu20=";
+    hash = "sha256-08Rl3vhXtlylNDc1gh5aZJ9/RoxeyxpDbklmhMVJuq4=";
   };
 
 in
@@ -37,10 +37,11 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   ];
 
   patches = [
-    # Plugin update to support Gradle 8.13; remove when included in a release.
+    # freeplane is using the wrong repository for a plugin
+    # remove when https://github.com/freeplane/freeplane/pull/2453 is merged and released
     (fetchpatch {
-      url = "https://github.com/freeplane/freeplane/commit/e58958783ef6f85ab00bf270c1f897093c4d7006.patch";
-      hash = "sha256-oQF/GbItl2ZEVlTKzojqk9xTWl8CVP7V3yig/py71hk=";
+      url = "https://github.com/amadejkastelic/freeplane/commit/973c49b7a73622e434bb86c8caea15383201b58a.patch";
+      hash = "sha256-iztFmISXZu8xKWqpwDYgBSl8ZSpZEtNriwM+EW1+s+Y=";
     })
   ];
 

